lib/src/component/dom_components.dart

```diff
@@ -910,3 +911,27 @@ abstract class Dom {
   /// optionally backed by a specified map.
   static SvgProps vkern([Map backingMap]) => SvgProps(react.vkern, backingMap);
 }
+
+class $DomPropsImpl extends DomProps with $DomPropsMixin {
```

Comment:
So is your thought process here that we'd manually author this class, as opposed to generating it? Or would this be the generated output we'd want?

If manually authoring it: looking at option 1 in the ticket, the idea behind using the props config is to avoid referencing the `$` members; otherwise, we may as well go with option 2.